src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((result, style) => {
+      const [property, value] = style
+        .split(':')
+        .map(part => part
+          .trim());
+
+      if (value) {
+        result[property] = value;
+      }
```

Comment:
I think it would be better to avoid some nested iterations in reduce. 

```suggestion
      const [property, value] = style.split(':');

      if (value) {
        result[property.trim()] = value.trim();
      }
```